### PR TITLE
Adjust colors, sizes of buttons group and buttons group dropdown

### PR DESCRIPTION
**Pull request by NeOMakinG**
Evo https://github.com/PrestaShop/PrestaShop/issues/16593

How to test : Read the readme of https://github.com/PrestaShop/prestashop-ui-kit and compare with informations in the #16593 issue

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1016,7 +1016,16 @@ <h3>Colored buttons</h3>
             <button type="button" class="btn btn-outline-danger">Danger</button>
             <button type="button" class="btn btn-warning">Warning</button>
             <button type="button" class="btn btn-outline-warning">Warning</button>
-        </article>
```
Comment (PierreRambaud) [inline index.html]: no more </article> ?

File: scss/_buttons.scss [hunk 2]
```diff
@@ -19,19 +19,40 @@
     font-size: 1.4em;
     margin-top: -0.083em;
   }
+
+  &-default {
+    @include ps-button-outline-variant(
+      #363a41,
```
Comment (PierreRambaud) [inline scss/_buttons.scss]: Must be a variable

File: scss/_buttons.scss [hunk 3]
```diff
@@ -19,19 +19,40 @@
...
+      $primary,
+      'default'
+    );
+
+    border-color: #dedede;
```
Comment (PierreRambaud) [inline scss/_buttons.scss]: same

File: scss/_buttons.scss [hunk 4]
```diff
@@ -19,19 +19,40 @@
...
+
+    &:hover {
+      border-color: #dedede;
+    }
+
+    &:not([disabled]):not(.disabled):active,
+    &:not([disabled]):not(.disabled).active {
+      color: white;
```
Comment (PierreRambaud) [inline scss/_buttons.scss]:
```suggestion
      color: $white;
```

File: scss/_buttons.scss (hunk 2 above)
Comment (PierreRambaud) [reply scss/_buttons.scss]: `$gray-700` or `$gray-800`

File: scss/_variables.scss [hunk 5]
```diff
@@ -31,6 +31,7 @@ $green:   #70b580;
 $orange:  #fd7e14 !default;
 $teal:    #20c997 !default;
 $cyan:    $blue;
+$light-blue: #F4FCFD;
```
Comment (PierreRambaud) [inline scss/_variables.scss]: lowercase

File: index.html [hunk 6]
```diff
@@ -1016,6 +1016,19 @@ <h3>Colored buttons</h3>
             <button type="button" class="btn btn-outline-danger">Danger</button>
             <button type="button" class="btn btn-warning">Warning</button>
             <button type="button" class="btn btn-outline-warning">Warning</button>
+
+            <br><br>
```
Comment (PierreRambaud) [inline index.html]: You must configure your IDE to remove unwanted whitespace at the end of lines.
```suggestion
            <br><br>
```

Comment (NeOMakinG) [reply index.html]: I added the conf and removed whitespaces

**Comment by PierreRambaud** [comment]
A rebase is needed

**Comment by eternoendless** [comment]
@NeOMakinG there's a conflict with `dist/css/bootstrap-prestashop-ui-kit.css`, can you rebase?

**Comment by eternoendless** [comment]
Thank you @NeOMakinG

**Pull request merged**